Gems/PhysX/Code/Include/PhysX/UserDataTypes.inl

```diff
@@ -23,38 +22,26 @@ namespace PhysX
         actor->userData = this;
     }
 
-    inline BaseActorData::BaseActorData(BaseActorData&& other) :
-        m_sanity(s_sanityValue), m_actorType(other.m_actorType), m_actor(AZStd::move(other.m_actor))
+    inline ActorData::ActorData(ActorData&& other)
+        : m_sanity(s_sanityValue)
```

Comment:
Shouldn't this assign `other.m_sanity` to `m_sanity`? Otherwise if it's moving from an invalid ActorData then suddenly the moved one would have different `m_sanity` and be valid. Same for the move operator.

Reply:
yeah, not sure it didn't do it before.. changed now,thanks!